docs/scripts_md/MakeNiiFilesBIDSCompliant.md

```diff
@@ -0,0 +1,143 @@
+# NAME
+
+MakeNIIFilesBIDSCompliant.pl -- a script that creates a BIDS compliant imaging
+dataset from the MINCs in the `assembly/` directory
+
+# SYNOPSIS
+
+perl tools/MakeNIIFilesBIDSCompliant.pl `[options]`
+
+Available options are:
+
+\-profile                : name of the config file in `../dicom-archive/.loris_mri`
+
+\-tarchive\_id            : The ID of the DICOM archive to be converted into BIDS
+                        dataset (optional, if not set, convert all DICOM archives)
+
+\-dataset\_name           : Name/Description of the dataset about to be generated
+                        in BIDS format; for example BIDS\_First\_Sample\_Data. The
+                        BIDS data will be stored in a directory called the `dataset_name`
+
+\-slice\_order\_philips    : Philips scanners do not have the `SliceOrder` in their
+                        DICOMs so provide it as an argument; `ascending` or
+                        `descending` is expected; otherwise, it will be logged
+                        in the JSON as `Not Supplied`"
+
+\-verbose                : if set, be verbose
+
+# DESCRIPTION
+
+This \*\*BETA\*\* version script will create a BIDS compliant NIfTI file structure of
+the MINC files currently present in the \`assembly\` directory. If the argument
+\`tarchive\_id\` is specified, only the images from that archive will be processed.
+Otherwise, all files in \`assembly\` will be included in the BIDS structure,
+while looping though all the 'tarchive\_id\`'s in the 'tarchive\` table.
```

Comment:
Are theses backticks need to be escape?

Reply:
Should be taken care of in the next commit if I don't screw it up ;)